channelmonitor/channelmonitor.go

```diff
@@ -416,14 +441,14 @@ func (mc *monitoredChannel) doRestartChannel() error {
 	}
 
 	// Send the restart message
-	log.Infof("%s: restarting (%d consecutive restarts)", mc.chid, restartCount)
+	log.Debugf("%s: restarting (%d consecutive restarts)", mc.chid, restartCount)
 	err := mc.sendRestartMessage(restartCount)
 	if err != nil {
 		log.Warnf("%s: restart failed, trying again: %s", mc.chid, err)
 		// If the restart message could not be sent, try again
 		return mc.doRestartChannel()
 	}
-	log.Infof("%s: restart completed successfully", mc.chid)
+	log.Debugf("%s: restart completed successfully", mc.chid)
```

Comment:
@dirkmc This should be Info. It'll only be once per successful debounced restart and will help us debug better.